### PR TITLE
Remove unused packages in docker images and host

### PR DESCRIPTION
**Pull request by qiluo-msft**
To reduce image size.
Unused packages include:
1. python3.4 inside all container: never used
2. libpython3.6 inside snmp container: not used and not required by any python packages

Fixed the build dependencies of python3.6

Use `apt-get purge` instead of `apt-get purge` to clean up even configuration files.

### CODE REVIEW

File: dockers/docker-snmp-sv2/Dockerfile.j2 [hunk 1]
```diff
@@ -60,7 +60,9 @@ RUN pip install /python-wheels/{{ whl }}
 RUN python3.6 -m sonic_ax_impl install
 
 # Clean up
-RUN apt-get -y purge libpython3.6-dev curl gcc make libdpkg-perl
+RUN apt-get -y purge libpython3.6-dev libpython3.6 curl gcc make libdpkg-perl
```
Comment (jleveque) [inline dockers/docker-snmp-sv2/Dockerfile.j2]: Should we move the `-y` after `purge` to align with all the other files while you're at it? (here and on line 65)?

Comment (qiluo-msft) [reply dockers/docker-snmp-sv2/Dockerfile.j2]: In my opinion, not necessary.

---
In reply to: [277117545](https://github.com/Azure/sonic-buildimage/pull/2807#discussion_r277117545) [](ancestors = 277117545)

Comment (jleveque) [reply dockers/docker-snmp-sv2/Dockerfile.j2]: Sure it's not necessary, just nice to unify everything.

**Comment by lguohan** [comment]
how much is reduced?

**Comment by qiluo-msft** [comment]
23.5 MB

**Comment by jleveque** [comment]
@qiluo-msft: Do you know if this will cherry-pick cleanly into the 201803 branch? If not, you will also need to create a similar PR there once this is approved and merged.

**Comment by qiluo-msft** [comment]
No clean cherry-pick. But should be a simple modification and I will submit a PR after this committed.

**Comment by lguohan** [comment]
can you provide more descriptions on the PR? not clear what you did in the PR.

**Pull request merged**